src/SpacetimeDBClient.cs

```diff
@@ -241,100 +241,75 @@ enum CompressionAlgos : byte
             Gzip = 2,
         }
 
-        private static BinaryReader BrotliReader(Stream stream)
+        private static BrotliStream BrotliReader(Stream stream)
         {
-            return new BinaryReader(new BrotliStream(stream, CompressionMode.Decompress));
+            return new BrotliStream(stream, CompressionMode.Decompress);
```

Comment:
At some point it might be nice to try and pool these, not sure it's possible, and not necessary now.